lib/Backends/OpenCL/OpenCL.h

```diff
@@ -187,7 +187,6 @@ class OCLBackend final : public Backend {
       case Kinded::Kind::MinNodeKind:
       case Kinded::Kind::MulNodeKind:
       case Kinded::Kind::QuantizeNodeKind:
-      case Kinded::Kind::ReluNodeKind:
```

Comment:
Please, investigate why it impacts the results. We have all needed support in OpenCL backend to allow this, there must be a bug ....

Reply:
sure, working on it